### ISSUE TITLE
Text Input Choice Field causes JSON Parse Error. 

### ISSUE THREAD

**Issue by james7342** [HotcakesCommerce/hotcakes-commerce-core#450]
## Describe the bug
Found this issue and am able to duplicate it with a Product Level and Shared Choice Text Input Box. 
(https://mmmcommerce.com/Forums/g/Posts/t/326/Shared-Choice-Text-Input-Field-BUG)

## Software Versions
- DNN:  9.11.2
- Hotcakes:  3.6.0

## To Reproduce
1. Create Choice item for Text Input - Shared or at Product Level 
2. Col. 1, Row 1, Size 50
3. View Product and enter text in input box. 
4. In browser console note: JSON Parse Error
5. Unable to add product to Cart.

If you remove the Text Input Size works as expected. 

## Expected behavior
No error and Product adds to Cart. 

## Actual behavior
Unable to add product to Cart. 

## Screenshots
JSON Error:
![screenshot_804](https://user-images.githubusercontent.com/27960723/231226440-dd480fc3-fd12-442f-9615-1a509da69781.png)

## Error log
**Note: Debug DLL's**
N/A

**Comment by WillStrohl** [comment]
Thank you for reporting this issue.  Our team will attempt to reproduce the issue and report back here once we're done.

**Comment by ismaelTartola** [comment]
@WillStrohl @james7342 I was able to reproduce the bug reported in this issue. I followed the same steps to reproduce. Only point that I think we should add a extra step to add the Shared Choice Item to the product. How that's sound? @james7342. These are my steps to reproduce proposal:

1.     Create Choice item for Text Input - Shared or at Product Level.
2.     Col. 1, Row 1, Size 50.
3.     Add the Created Shared Choice to a Product.
4.     View Product and enter text in input box.
5.     In browser console note: JSON Parse Error.
6.     Unable to add product to Cart.

Regards,

**Comment by james7342** [comment]
@ismaelTartola, Yes, I didn’t specially call out adding the shard item step. It effects both and my thought was if it was fixed at the product level it would probably address the shared aspect of it also. Both options would have to be verified I think before the issue could be closed.
